esp/templates/survey/answers.html

```diff
@@ -59,6 +54,28 @@
                     <li>Average for all classes: {{ q.question.global_average }}</li>
                 {% endif %}
             </ul>
+            <!-- Creates a key that provides details about the Histogram Labeling-->
+            <b>Response Labels:</b>
+            {% for label in q.question.get_params.list %}
+            <ul>
+                <li> <b> {{ forloop.counter }}: </b>{{ label }} </li>
+            </ul>
+            {% endfor %}
```

Comment:
The `<ul>` tags should go outside the forloop, otherwise it looks like this:
![image](https://user-images.githubusercontent.com/7232514/88408216-da55ca80-cd98-11ea-942f-61df523e1ed6.png)
```suggestion
            <ul>
            {% for label in q.question.get_params.list %}
                <li> <b> {{ forloop.counter }}: </b>{{ label }} </li>
            {% endfor %}
            </ul>
```

Reply:
Once that is done, I think we'll want to wrap the whole thing in an if loop to make sure the list exists (otherwise you get an empty `<ul>` that takes up a little space).

Reply:
When I was testing this even when the list doesn't have labels, the list still has number_of_ratings amount of objects in the list (even if they are empty). Do you want me to NOT display anything if number_of_ratings is zero? The problem is that the list won't be empty because number_of_ratings denotes how many items there are in the list even if the user doesn't give them labels.

Reply:
Oh, I was mostly talking about when the question is of type numeric rating, we want to skip this part (and not include an empty `<ul>`), and vice versa when the question is of type labelled numeric rating (see my comment below).